modules/riegeli/0.0.0-20240605-43effad/presubmit.yml

```diff
@@ -0,0 +1,49 @@
+matrix:
+  platform:
+  - debian10
+  - ubuntu2004
+  bazel:
+  - 6.x
+  - 7.x
+tasks:
+  verify_cpp:
+    name: Verify C++ targets
+    platform: ${{ platform }}
+    bazel: ${{ bazel }}
+    build_flags:
+    - '--cxxopt=-std=c++17'
+    build_targets:
+    - '@riegeli//riegeli/base/...'
+    - '@riegeli//riegeli/brotli/...'
+    - '@riegeli//riegeli/bytes/...'
+    - '@riegeli//riegeli/bzip2/...'
+    - '@riegeli//riegeli/chunk_encoding/...'
+    - '@riegeli//riegeli/containers/...'
+    - '@riegeli//riegeli/csv/...'
+    - '@riegeli//riegeli/digests/...'
+    - '@riegeli//riegeli/endian/...'
+    - '@riegeli//riegeli/lines/...'
+    - '@riegeli//riegeli/lz4/...'
+    - '@riegeli//riegeli/messages/...'
+    - '@riegeli//riegeli/ordered_varint/...'
+    - '@riegeli//riegeli/records:record_position'
```

Comment:
Can we build `records:all` here?

Reply:
No because we have the `py_proto_library()` in there.

Reply:
Awesome, I updated to the new version and am using `:all` now. Thanks!